functions/Invoke-DbaLogShipping.ps1

```diff
@@ -664,7 +664,7 @@ The script will show a message that the copy destination has not been supplied a
 		}
 
 		# Check the backup network path
-		if (-not ((Test-Path $BackupNetworkPath -PathType Container -IsValid -Credential $SourceCredential) -and (Get-Item $BackupNetworkPath).PSProvider.Name -eq 'FileSystem')) {
+		if ((Test-DbaSqlPath -Path $BackupNetworkPath -SqlInstance $SourceSqlInstance -SqlCredential $SourceCredential) -ne $true){
```

Comment:
There will be situations where SQL Server account will not have access to certain directories. In most cases the service account that needs the access is the agent service account.

As I'm writing this I'm thinking we probably need to supply the user with information to check the directories after they're created. The dba/user should grant the agent service account the necessary privileges to the just created directories, otherwise the log shipping will not work..
In normal circumstances you use directories that are already being used by sql server or you've already set the privileges. 

I agree with the comment to make sure that in case the Test-DbaSqlPath does not work, we still try to create the directory in the original way using Invoke-Command2.